src/controllers/controlpickermenu.cpp

```diff
@@ -354,46 +354,53 @@ ControlPickerMenu::ControlPickerMenu(QWidget* pParent)
 
     // Library Controls
     QMenu* libraryMenu = addSubmenu(tr("Library"));
-    addPrefixedControl("[Playlist]", "ToggleSelectedSidebarItem",
-                       tr("Expand/Collapse View"),
-                       tr("Expand/collapse the selected view (library, playlist..)"),
+    addPrefixedControl("[Library]", "MoveUp",
+                       tr("Move up"),
+                       tr("Equivalent to pressing the UP key on the keyboard"),
                        m_libraryStr, libraryMenu);
-
-    addPrefixedControl("[Playlist]", "SelectPlaylist",
-                       tr("Switch Next/Previous View"),
-                       tr("Switch to the next or previous view (library, playlist..)"),
+    addPrefixedControl("[Library]", "MoveDown",
+                       tr("Move down"),
+                       tr("Equivalent to pressing the DOWN key on the keyboard"),
+                       m_libraryStr, libraryMenu);
+    addPrefixedControl("[Library]", "MoveVertical",
+                       tr("Move up/down"),
+                       tr("Move vertically in either direction using a knob, as if pressing UP/DOWN keys"),
+                       m_libraryStr, libraryMenu);
+    addPrefixedControl("[Library]", "ScrollUp",
+                       tr("Scroll Up"),
+                       tr("Equivalent to pressing the PAGE UP key on the keyboard"),
                        m_libraryStr, libraryMenu);
-    addPrefixedControl("[Playlist]", "SelectNextPlaylist",
```

Comment:
> We can't remove these old ones before we have no intuitive solution for a RMX2 style control buttons

As we previously discussed, my understanding is that currently UP/DOWN maps to `slotSelectTrack()`, and LEFT/RIGHT maps to `slotSelectSidebarItem()`. Furthermore, there are two buttons LoadA and LoadB that load the currently selected track into Deck A / Deck B respectively. Is this correct?

If this is correct then the scheme that I previously suggested is perfectly intuitive. The left and right buttons will change the currently focused pane (shift+tab, tab on keyboard), and the up and down buttons will change the selected item within the currently focused pane (up, down buttons on keyboards). This is FAR more intuitive than using left and right to navigate up and down inside the left sidebar  IMHO.

Shift+Left, Shift+Right buttons should be mapped to Left, Right keys on the keyboard
Shift + Up, Shift + Down buttons should be mapped to PgUp, PgDown keys on keyboard

If you feel this is unintuitive then please explain yourself in detail.